bluebrain/repo-bluebrain/packages/py-bcsb/package.py

```diff
@@ -13,7 +13,7 @@ class PyBcsb(PythonPackage):
     git = "ssh://git@bbpgitlab.epfl.ch/viz/brayns/braynscircuitstudiobackend.git"
 
     version("develop", branch="develop")
-    version("2.4.0", tag="v2.4.0")
+    version("2.4.1", tag="v2.4.1")
```

Comment:
That tag does not seem to exist.

Reply:
My bad ^^'

Reply:
Could you retrigger the CI now that I created the tag?

Reply:
Fixed thanks.

Just need you blessing now ;)

Reply:
There you go :)